src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -152,7 +153,7 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 			String line;
 	
 			try {
-				br = new BufferedReader(new FileReader(tempOutfilename));
+				br = new BufferedReader(new InputStreamReader(new FileInputStream(tempOutfilename),"UTF-8"));

```

Comment:
Could be used `Files.newBufferedReader` and `StandardCharsets.UTF_8` instead, it's easier to read.